canonical/ubuntu.com
# WD-8594 - remove /download/kvm page

## Done

Remove /download/kvm page and redirect to /core/docs/getting-started

## QA
- [demo link](https://ubuntu-com-13523.demos.haus/download/kvm)
- [copy doc](https://docs.google.com/document/d/1FHIArLpuAyLe2gnDwymefvzc4Lg8dSGvN8QAl8wX18Q/edit#heading=h.y5kr0riv2frn)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is remove and redirected as expected

## Issue / Card
[WD-8594](https://warthogs.atlassian.net/browse/WD-8594)

Fixes #

## Screenshots


[WD-8594]: https://warthogs.atlassian.net/browse/WD-8594?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
